skimage/transform/_thin_plate_splines.py

```diff
@@ -97,51 +97,122 @@ def _make_inverse_warp(
     return transform
 
 
-_small = 1e-100
-
-
 def _U(x):
-    return (x**2) * np.where(x < _small, 0, np.log(x))
+    """Compute basis kernel function for thine-plate splines.
 
+    Parameters
+    ----------
+    x: ndarray
+        Input array representing the norm distance between points.
+        The norm is the Euclidean distance.
+    Returns
+    -------
+    ndarray
+        Calculated U values.
+    """
+    _small = 1e-8  # Small value to avoid divide-by-zero
```

Comment:
Hmm, I am curious about why you decided to change the `_small` value.

Reply:
You mean where it is declared or the value itself?
The value itself can be any small value. The variable isn't used in other functions, it appropriate the declare inside the function for closure and reduce noise in the global namespace.